src/main/resources/templates/about.html

```diff
@@ -144,11 +144,11 @@ <h3>Technologies</h3>
                     </div>
                     <div class="col-md-4">
                         <p>JQuery
-                          <a class="link" aria-label="Link to JQuery API" th:href="#{groupings.technologies.url.jquery_api_current_version}">(Current Version)</a>
+                          <a class="link" aria-label="(Current Version)" th:href="#{groupings.technologies.url.jquery_api_current_version}">(Current Version)</a>
                           <a class="link" aria-label="Link to JQuery API" th:href="#{groupings.technologies.url.jquery_api}">(API)</a>
                         </p>
                         <p>Jasmine
-                          <a class="link" aria-label="Link to Jasmine Guide" th:href="#{groupings.technologies.url.jasmine_guide_current_version}">(Current Version)</a>
+                          <a class="link" aria-label="(Current Version)" th:href="#{groupings.technologies.url.jasmine_guide_current_version}">(Current Version)</a>
```

Comment:
I think it would more descriptive if the aria-labels said "Link to JQuery version" and "Link to Jasmine version", unless the ADA tool you were using requires it to match the text.

Reply:
I agree but yes the ADA tool is flagging to match the text. Alternatively I could instead change the text to match the labels to keep them descriptive if we wanted to do that instead.

Reply:
Ok that's fine to leave it as is.